tools/__tasks__/compile/fonts/index.config.js

```diff
@@ -1,435 +1,507 @@
     {
-        dest: 'GuardianTextEgyptianWebHinted.woff.json',
-        'font-family': '"Guardian Text Egyptian Web"', // TODO: Rename to Guardian Text Egyptian?
+        dest: 'GuardianTextEgyptianHinted.woff.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Regular.woff'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Regular.woff',
             },
             {
                 'font-style': 'italic',
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-RegularItalic.woff'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-RegularItalic.woff',
             },
             {
                 'font-weight': '700',
                 'font-style': 'italic',
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Bold.woff'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Bold.woff',
             },
         ],
     },
-    // *** Guardian Text Egyptian - ttf - Hinted ***    
+    // *** Guardian Text Egyptian - ttf - Hinted ***
     {
-        dest: 'GuardianTextEgyptianWebHinted.ttf.json',
-        'font-family': '"Guardian Text Egyptian Web"', // TODO: Rename to Guardian Text Egyptian?
+        dest: 'GuardianTextEgyptianHinted.ttf.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
                 'font-weight': '300',
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Regular.ttf'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Regular.ttf',
             },
             {
                 'font-weight': '300',
                 'font-style': 'italic',
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-RegularItalic.ttf'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-RegularItalic.ttf',
             },
             {
                 'font-weight': '700',
                 'font-style': 'italic',
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Bold.ttf'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Bold.ttf',
             },
         ],
     },
     // *** Guardian Text Sans - woff2 - Not Hinted ***
     {
-        dest: 'GuardianTextSansWeb.woff2.json',
-        'font-family': '"Guardian Text Sans Web"', // TODO: Rename to Guardian Text Sans?
+        dest: 'GuardianTextSans.woff2.json',
+        'font-family': '"Guardian Text Sans"',
         fonts: [
             {
-                src: 'GuardianTextSans/noalts-not-hinted/GuardianTextSans-Regular.woff2'
+                src:
```

Comment:
Font weight numbers similarly missing? (maybe this will fix TextSans Bold replacing Regular on CODE?)